app/src/main/java/com/zeapo/pwdstore/PasswordStore.kt

```diff
@@ -127,7 +127,13 @@ class PasswordStore : AppCompatActivity(R.layout.activity_pwdstore) {
                     return@registerForActivityResult
                 }
             }
-            val intent = Intent(activity, GitOperationActivity::class.java)
```

Comment:
This is the 'real' fix, we were swallowing the clone action because `GitServerConfigActivity` now handles it.